### ISSUE TITLE
[FR] navigation : locate enlarged area centered to mouse cursor (not to whole image as is currently the case)

### ISSUE THREAD

**Issue by pehar1** [darktable-org/darktable#11655]
An example to clarify what is meant and why this would be useful :

- Go to darkroom
- Select _navigation module -> fit to screen_
- Select _retouch_ module, do some retouches, for example near the top left corner of the image

Now you notice that you have some fine detail where you have to place the retouches with very high precision. The mouse cursor is still placed around the position where you want to continue your work.

- Press _ALT-1_ to zoom in (100%). The enlarged area shown now is always centered relative to the whole image. You have to click/drag/move around to search the area again (left top corner) where you want to continue your work. This procedure has to be repeated every time you switch between _ALT-3_ (fit to screen) and _ALT-1_ (100%).

It would be very useful to center the enlarged area (brighter area in navigation module) around the mouse location relative to the image area. Only if the mouse is located inside the image area of darkroom, of course. This would save a lot of mouse movements and digging around, work could be continued nearly immediatly.

**Comment by ralfbrown** [comment]
Using the mouse scroll wheel instead of a keyboard shortcut already works the way you want.

**Comment by pehar1** [comment]
Reopend the feature request because I did several further edits on reproductions (slides, negatives) where the retouch module has to be used in "CTRL-circle" mode to allow continuous setting of shapes. In this case the use of the mouse wheel does **not** zoom into the image but changes the size of the shape, "CTRL-mouse wheel" changes the opacity and "SHIFT-mouse" wheel changes the feather size.
I did not find any way to zoom in, except using the keyboard shortcut "ALT-1" or using the pulldown menu in navigation module with the problems described above.
I continue to believe that it would be very useful to center the enlarged area around the mouse location, even if one uses the shortcuts to zoom in/out.
